### DISCUSSION TITLE
Native support password hash

### DISCUSSION THREAD

**Discussion by baek-sang** [dotnet/runtime#65945] [Ideas]
bcrypt, argon2 etc..

Don't want using external libarary

**Comment by gfoidl** [comment]
If you want this to be added / considered to be added please create an issue with the [api suggestion](https://github.com/dotnet/runtime/blob/main/docs/project/api-review-process.md).

**Reply by Xyncgas** [reply]
Or you can get a nuget package.
if .NET implements this, it would likely come in as a nuget package too.
especially, this is application, .NET is a computing language platform

**Comment by huoyaoyuan** [comment]
#15936 #19933 #39922
